src/Foundation/NSMetadataItem.cs

```diff
@@ -24,6 +24,16 @@ bool GetBool (NSString key)
 			return n == null ? 0 : n.DoubleValue;
 		}
 
+		nint GetNInt (NSString key)
+		{
+			var n = Runtime.GetNSObject<NSNumber> (GetHandle (key));
+#if XAMCORE_2_0
+			return n?.NIntValue ?? 0;
```

Comment:
Is `0` if `nil` documented as default value? maybe we should use a nullable here if not